### ISSUE TITLE
Incompatibility with eslint v4

### ISSUE THREAD

**Issue by fvictorio** [Gillespie59/eslint-plugin-angular#490]
It seems like 6a0111dd259b2a8eb396a10682144e855bd90d69, which upgrades the dependency to version 4 of eslint, broke the build. From what I see, the problem is [this](http://eslint.org/docs/user-guide/migrating-to-4.0.0#-ruletester-now-validates-properties-of-test-cases).

I'm not familiar enough with the codebase to propose a solution, except "downgrade to v3". It seems like custom properties in test cases are used a lot, so fixing it it's not trivial, but let me know if I can help in some way.

**Comment by Gillespie59** [comment]
Yes I am aware of that problem :s The problem is related to the tests we are running on our documentation I think.

**Comment by Gillespie59** [comment]
For the moment, I have just commented the testDocs method in the gulpfile config file. 
When I remove the valid property from the test option, some tests failed...
@tilmanpotthof do you have some time to have a look to this documentation tests ?
